### PR TITLE
 Readme

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
merge when octobot.cloud is up with /trading-bot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,24 +32,26 @@ We are looking forward to receiving your feedback on our new OctoBot based syste
...
 - [Contribute](https://www.octobot.cloud/en/guides/octobot-developers-environment/setup-your-environment?utm_source=github&utm_medium=dk&utm_campaign=regular_open_source_content&utm_content=contribute) to improve OctoBot core repositories and tentacles.
 
-Looking for more info ? Check out our Octobot guides at [octobot.cloud/guides](https://www.octobot.cloud/en/guides/?utm_source=github&utm_medium=dk&utm_campaign=regular_open_source_content&utm_content=checkout_guides)
+OctoBot is **AI ready**: Python being the main language for OctoBot, it's easy to integrate machine-learning libraries such as [Tensorflow](https://github.com/tensorflow/tensorflow) or any other libraries and take advantage of all the available data and create a very powerful trading strategy.
```
Comment (Herklos) [inline README.md:48]: Should we add a link the the AI feature page ?

Comment (GuillaumeDSM) [reply README.md:48]: Good idea !

File: README.md [hunk 2]
```diff
@@ -40,24 +32,26 @@ We are looking forward to receiving your feedback on our new OctoBot based syste
...
 - [Contribute](https://www.octobot.cloud/en/guides/octobot-developers-environment/setup-your-environment?utm_source=github&utm_medium=dk&utm_campaign=regular_open_source_content&utm_content=contribute) to improve OctoBot core repositories and tentacles.
 
-Looking for more info ? Check out our Octobot guides at [octobot.cloud/guides](https://www.octobot.cloud/en/guides/?utm_source=github&utm_medium=dk&utm_campaign=regular_open_source_content&utm_content=checkout_guides)
+[OctoBot is AI ready](https://www.octobot.cloud/features/ai-trading-bot?utm_source=github&utm_medium=dk&utm_campaign=regular_open_source_content&utm_content=ai-ready): Python being the main language for OctoBot, it's easy to integrate machine-learning libraries such as [Tensorflow](https://github.com/tensorflow/tensorflow) or any other libraries and take advantage of all the available data and create a very powerful trading strategy.
```
Comment (Herklos) [inline README.md:48]: :+1:

**Review by Herklos — APPROVED** [review]
"waiting" :smile:

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Comment by GuillaumeDSM** [comment]
merge ?

**Comment by Herklos** [comment]
> merge ?

yes

**Pull request merged**